### ISSUE TITLE
Error when following Installation Docs - CDN, HLS, Default Styling, and Add Player Markup

### ISSUE THREAD

**Issue by trueChazza** [vidstack/player#780]
### Current Behavior:

Following [Installation Docs](https://www.vidstack.io/docs/player/getting-started/installation/cdn/hls?styling=defaults) for CDN, HLS, Default Styling, and Add Player Markup - I receive the following browser console error:

<img width="743" alt="Screenshot 2023-02-18 at 2 56 01 PM" src="https://user-images.githubusercontent.com/20368025/219826628-27344828-b6bf-4e7e-b3bd-ccce07958b71.png">

`Uncaught TypeError: Failed to resolve module specifier "media-icons". Relative references must start with either "/", "./", or "../".`

No video element rendered on page.

### Expected Behavior:

Following [Installation Docs](https://www.vidstack.io/docs/player/getting-started/installation/cdn/hls?styling=defaults) for CDN, HLS, Default Styling, and Add Player Markup - I should receive no browser console error.

Video element should render on page

### Steps To Reproduce:

Follow [Installation Docs](https://www.vidstack.io/docs/player/getting-started/installation/cdn/hls?styling=defaults) for CDN, HLS, Default Styling, and Add Player Markup.

### Root Cause:

<img width="1512" alt="Screenshot 2023-02-18 at 3 10 33 PM" src="https://user-images.githubusercontent.com/20368025/219826451-4f127ab0-809e-4284-ab7f-31f8deeaa2df.png">

After some investigation I was able to fix this by having `media-icons` import resolve to [https://cdn.jsdelivr.net/npm/media-icons/dist/browser/index.js](https://cdn.jsdelivr.net/npm/media-icons/dist/browser/index.js)

I'm aware this is not where it should resolve to. Resolving to the correct path would fix this issue.

<img width="1512" alt="Screenshot 2023-02-18 at 3 19 41 PM" src="https://user-images.githubusercontent.com/20368025/219826707-17b31809-fc8c-4814-a871-19ca96d58500.png">

After correcting import path I received no browser console error, with video element rendered.

### Environment:

- Framework: NA
- Meta Framework: NA
- Device: MacBook
- OS: MacOS 13.0 Ventura
- Browser: Chrome 109

### Anything Else?

Thanks for your work on this library, it's been exciting watching the progress and I look forward to using this in production as soon as possible 🎸

**Comment by mihar-22** [comment]
Thanks @trueChazza, really appreciate the kind words :) This has been fixed in `0.3.2`, you might need to manually pin the CDN URL if JSDelivr doesn't update straight away like so:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vidstack@0.3.2/styles/base.min.css" />
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vidstack@0.3.2/styles/ui/buttons.min.css" />
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/vidstack@0.3.2/styles/ui/sliders.min.css" />
<script type="module" src="https://cdn.jsdelivr.net/npm/vidstack@0.3.2/dist/cdn/prod.js"></script>
```

You can first test without pinning the version, if it fails pin the version and test removing it in a day or so.

**Comment by trueChazza** [comment]
Awesome, thank you!
